DESCRIPTION

```diff
@@ -1,9 +1,13 @@
 Package: tidylog
 Type: Package
-Title: Logging for 'dplyr' and 'tidyr' Functions
+Title: Logging for 'dplyr' and 'tidyr' functions
```

Comment:
not a typo

Reply:
Done: 3e70eeabf45726c20c053d5eb159d3ffc2e5de38